### DISCUSSION TITLE
Lesson 7: error in DevOpsTools.sol

### DISCUSSION THREAD

**Discussion by yasharyan0904** [Cyfrin/foundry-full-course-cu#1782] [Q&A]
After running the following code:
```
forge install Cyfrin/foundry-devops --no-commit
```
It gets installed but there is not file like get_recent_deployment.sh , instead StringUtils.sol file is there below the DevOpsTools.sol file.

Also, DevOpsTools.sol file is showing the following error:
<img width="868" alt="CaptureS__" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/144695005/3104a47f-fbed-4b98-bbb9-63aba6a82eb1">

**Comment by EngrPips** [comment]
Hello @yasharyan0904, Will check with Cyfrin to see what could be wrong.

**Reply by yasharyan0904** [reply]
Any updates?

**Reply by EngrPips** [reply]
I haven't received an update yet. Could you please try what @BLTC-520  suggested? Let's see if that helps, at least for now.

**Comment by BLTC-520** [comment]
Same question here.

**Reply by BLTC-520** [reply]
I did see the function in forge-std/src/Vm.sol at line 550. 

```
/// Checks if `key` exists in a JSON object.
    function keyExists(string calldata json, string calldata key) external view returns (bool);
```

.keyExistsJson didn't exist in Vm.sol. So, I use vm.keyExists instead of vm.keyExistsJson, and it can run successfully. @yasharyan0904 Hope it helps.

**Reply by EngrPips** [reply]
Thanks for the suggestion, friend.

**Reply by BLTC-520** [reply]
You're welcome 🫡
